src/main/java/io/reactiverse/pgclient/PgConnectOptions.java

```diff
@@ -48,6 +51,15 @@
 
   public PgConnectOptions() {
     super();
+    if (getenv("PGHOSTADDR") == null) {
+      host = getenv("PGHOST") == null ? DEFAULT_HOST : getenv("PGHOST");
+    } else {
+      host = getenv("PGHOSTADDR");
+    }
+    if (getenv("PGPORT") != null) port = parseInt(getenv("PGPORT"));
```

Comment:
curly braces

Reply:
should catch NumberFormatException to use the default value in case

Reply:
Fixed in https://github.com/reactiverse/reactive-pg-client/pull/87/commits/2d25295cca19ce3f8217f1d7995a0d8f80cdd69a.